en/asgardeo/mkdocs.yml

```diff
@@ -537,7 +539,7 @@ nav:
       - Identity recovery API: apis/organization-apis/org-account-recovery.md
       - Idle account identification API: apis/organization-apis/org-idle-account-identification.md
       - Invite parent organization's users API: apis/organization-apis/organization-user-invitation.md
-      - Notification sender API: apis/organization-apis/notification-sender-rest-api.md
```

Comment:
this will cause existing link wso2.com/asgardeo/docs/apis/organization-apis/notification-sender-rest-api to be broken, won't it? shall we add a redirection to avoid that?

Reply:
already added with https://github.com/wso2/docs-is/pull/5320/files#diff-a325268ca0b4795eee063a30248ca59cbfe240010598191ee99da58ab02a0b29R158-R159